src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Ui {
+
+    static String commandNotFound = "Oops! Command not found!";
+    static String taskNeedsName = "Oops! This task needs a name!";
+    static String taskNeedsDateTime = "Oops! This task needs a date/time!";
+    static String wrongDateTimeFormat = "Oops! The date/time needs to be DD/MM/YYYY HHMM (24hr) format!";
+    static String noTaskFound = "Oops! Task not found in the list!";
+    static String noTaskInList = "Yay! There are no tasks in your list!";
+    static String displayTaskList = "Here are your tasks in your list:";
+    static String displayMatchingTaskList = "Here are your tasks that matched your query:";
+    static String noMatchingTaskInList = "Oops! There are no tasks that matches your query!";
+    static String changesSaved = "Changes saved!";
+    static String markedTaskDone = "Marked this task as done:\n    ";
+    static String taskAlreadyDone = "Oops! This task was already marked as done!";
+    static String newTaskMessage = "New task has been added:\n    ";
+    static String deleteTaskMessage = "Removed this task:\n    ";
+    static String welcomeMessage = "-------------------------------------\nWelcome! What can I do for you today?";
+    static String exitMessage = "Thank you for using Duke.\nHave a nice day!\n";
```

Comment:
I'd suggest that all these preset messages have their names be written in SCREAMING_SNAKE_CASE and made `final` to show that they are constants!

For e.g.: `public static final String COMMAND_NOT_FOUND_MESSAGE = "Oops! ...";`